core/src/i18n/en.yaml

```diff
@@ -117,7 +119,65 @@ discovery-and-network:
 functions:
   title: Functions
   variable:
-    tooltip: This variable comes from a Resource. To get the value, follow the link.
+    buttons:
+      add: Add environment variable
+      open:
+        custom: Custom Variable
+        secret: Secret Variable
+        config-map: Config Map Variable
+    errors:
+      empty: Variable is empty.
+      duplicated: Duplicated variable name.
+      invalid: Variable name must contain alphanumeric characters, can contain '_', no blank spaces.
+      invalid-config: Select the config map name and key
+      invalid-secret: Select the secret name and key
+      restricted: This variable name is restricted and cannot be used. Try a different one.
+    form:
+      config-map: $t(functions.variable.type.config-map)
+      key: $t(functions.variable.header.key)
+      name: Name
+      secret: $t(functions.variable.type.secret)
+      value: $t(functions.variable.header.value)
+    header:
+      key: Key
+      name: Variable Name
+      source: Source
+      value: Value
+    not-found: This Function doesn't have any environment variables yet.
+    not-match: Couldn't find environment variables matching this query.
+    title:
```

Comment:
Having it in the order `variable.title.create-modal...` seems like it's the variable's title, but it's the modal title. I would switch the order to `variable.create-modal.title...`.